src/azure-cli/azure/cli/command_modules/resource/_params.py

```diff
@@ -93,6 +92,10 @@ def load_arguments(self, _):
         c.argument('scope', help='Fully-qualified scope for retrieving links.')
         c.argument('filter_string', options_list=['--filter', c.deprecate(target='--filter-string', redirect='--filter', hide=True)], help='Filter string for limiting results.')
 
+    with self.argument_context('resource tag') as c:
+        c.argument('is_incremental', options_list=['--is-incremental', '-i'], arg_type=get_three_state_flag(),
```

Comment:
I think 'store_true' may be more suitable  because we will not start  incremental tag by default.

Reply:
Good suggestion, changed it